packages/dds/map/src/map.ts

```diff
@@ -332,9 +332,8 @@ export class SharedMap extends SharedObject<ISharedMapEvents> implements IShared
     protected async loadCore(storage: IChannelStorageService) {
         const header = await storage.read(snapshotFileName);
 
-        const data = fromBase64ToUtf8(header);
         // eslint-disable-next-line @typescript-eslint/ban-types
-        const json = JSON.parse(data) as object;
+        const json = JSON.parse(fromBase64ToUtf8(header)) as object;
```

Comment:
I believe fromBase64ToUtf8 should be removed here, after read() call is replaced with readString()